versions.md

```diff
@@ -90,6 +90,8 @@ Forthcoming
 * Setting vendor_id in the received CacheChange_t for Data and DataFrag.
 * Added new DynamicData to JSON serializer (`json_serialize`).
 * Added new DynamicType to IDL serializer (`idl_serialize`).
+* Refactor discovery info structures
+  * Participant discovery data refactor
```

Comment:
Remove this and add:
1. One line similar to line 20 in line 21
2. Two line in line 25 with `ParticipantProxyData` and `ParticipantDiscoveryInfo`